src/main/scala/org/broadinstitute/dsde/firecloud/model/WorkspaceName.scala

```diff
@@ -0,0 +1,70 @@
+package org.broadinstitute.dsde.firecloud.model
+
+import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+
+import scala.annotation.meta.field
+
+@ApiModel(value = "Workspace Name")
+case class WorkspaceName  (
+                        @(ApiModelProperty@field)(required = true, value = "The namespace the workspace belongs to")
+                        namespace: Option[String] = None,
+                        @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
+                        name: Option[String] = None)
+
+// TODO: Revisit this if the rawls service removes createdDate and createdBy from their API.
+@ApiModel(value = "Workspace Entity")
+case class WorkspaceEntity  (
+                              @(ApiModelProperty@field)(required = true, value = "The namespace the workspace belongs to")
+                              namespace: Option[String] = None,
+                              @(ApiModelProperty@field)(required = true, value = "The name of the workspace")
+                              name: Option[String] = None,
+                              @(ApiModelProperty@field)(required = true, value = "The date the workspace was created in yyyy-MM-dd'T'HH:mm:ssZZ format")
+                              createdDate: Option[String] = None,
+                              @(ApiModelProperty@field)(required = true, value = "The user who created the workspace")
+                              createdBy: Option[String] = None,
+                              @(ApiModelProperty@field)(required = true, value = "The attributes of the workspace")
+                              attributes: Option[Map[String, String]] = None)
+
+case class EntityCreateResult(entityType: String, entityName: String, succeeded: Boolean, message: String)// TODO: This is a stub case class until we know what we're returning from the batch entity create endpoint.
```

Comment:
It's tough to review this with the long lines. Would you please wrap them at 100? To make it less painful, I suggest you use a two-space indent for the `@(ApiModelProperty@field)` values.